packages/framework/fluid-framework/package.json

```diff
@@ -12,8 +12,10 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "concurrently npm:build:compile npm:lint && npm run build:docs",
-    "build:compile": "npm run tsc",
+    "build:commonjs": "npm run tsc",
+    "build:compile": "concurrently npm:build:commonjs npm:build:esnext",
```

Comment:
this concurrently syntax is new to me, but I LOVE it.  All the escaping made me hate setting it up. This is super smooth.